packages/build-lz-options/src/utilities/prompts.ts

```diff
@@ -93,6 +93,14 @@ const promptForNativeDropAddress: PromptObject<'nativeDropAddress'> = {
     name: 'nativeDropAddress',
     message: 'What native gas drop do you want to set?',
     initial: makeBytes32(),
+    validate: (str: string) => {
+        try {
+            makeBytes32(str)
```

Comment:
Addresses are 20bytes long, it might be the case that some strings pass this validation but are not valid addresses; do we have a makeBytes20 or makeAddress() ?

Reply:
Different addresses can be different lengths.  For example:
- bitcoin is 32
- aptos is 32
- evm is 20